Nodejs/Product/TestAdapter/TestFrameworks/Jest/jest.js

```diff
@@ -50,15 +56,17 @@ const run_tests = function (context) {
                 fullyQualifiedName: testCase.fullyQualifiedName
             });
         }
-
-        const config = {
+        
+        let config = readConfigs(projectFolder, context);
+        
+        const argv = {
             json: true,
             reporters: [[__dirname + '/jestReporter.js', { context }]],
-            testMatch: [context.testCases[0].testFile]
-        };
+            config : JSON.stringify(config)
```

Comment:
I'm assuming putting `testMatch` in the config has the same behavior as having it outside (like before)?